contracts/Boilerplate/Catalogue.sol

```diff
@@ -53,3 +62,35 @@ contract Catalogue {
         config = _config;
     }
 }
+
+
+contract StorageCatalogue {
```

Comment:
I would definitely put it into a separate file. The StorageCatalogue is expected to be updated far less often then the Catalogue.